ui/selectors/selectors.js

```diff
@@ -3152,26 +3157,64 @@ export function getIsWatchEthereumAccountEnabled(state) {
   return state.metamask.watchEthereumAccountEnabled;
 }
 
+///: BEGIN:ONLY_INCLUDE_IF(bitcoin)
 /**
- * Get the state of the `bitcoinSupportEnabled` flag.
+ * Check if bitcoinAccounts feature flag is enabled with proper version check.
+ * Uses shared multichain feature flag utility - no duplicate logic.
+ *
+ * @param {unknown} flagValue - The feature flag value
+ * @returns {boolean} True if flag is enabled and meets minimum version requirement
+ */
+export function isBitcoinAccountsFlagEnabled(flagValue) {
+  return isBitcoinFeatureEnabled(flagValue);
+}
+
+///: END:ONLY_INCLUDE_IF
+
+/**
+ * Get the state of the `bitcoinSupportEnabled` flag with version check.
+ * Uses bitcoinAccounts feature flag.
  *
  * @param {*} state
  * @returns The state of the `bitcoinSupportEnabled` flag.
  */
 export function getIsBitcoinSupportEnabled(state) {
-  const { addBitcoinAccount } = getRemoteFeatureFlags(state);
-  return Boolean(addBitcoinAccount);
+  ///: BEGIN:ONLY_INCLUDE_IF(bitcoin)
+  const { bitcoinAccounts } = getRemoteFeatureFlags(state);
+  return isBitcoinAccountsFlagEnabled(bitcoinAccounts);
```

Comment:
IMO, `isBitcoinAccountsFlagEnabled` should not expect the `bitcoinAccounts` object directly.

Instead, we should pass-in the `getRemoteFeatureFlags(state)` object, and `isBitcoinAccountsFlagEnabled` would use `bitcoinAccounts` from it (inside `isBitcoinAccountsFlagEnabled` I mean).

Because right now, I could be passing the result of `solanaAccounts` to `isBitcoinAccountsFlagEnabled` and that's ok, and I think it defeats the purpose of having named functions like this.

WDYT?